add a note to secrets page (#2541)
pulumi/pulumi-hugo: themes/default/content/docs/intro/concepts/secrets.md

    weight: 7
---

> All resource input and output values are recorded as [`state`](/docs/intro/concepts/state), and are stored in the Pulumi Service, a file, or a pluggable provider that you choose. These raw values are usually just server names, configuration settings, and so on. In some cases, however, these values contain sensitive data, such as database passwords or service tokens. **Pulumi never sends any authentication secrets or credentials to the Pulumi service**.

Author: ```suggestion
All resource input and output values are recorded as [`state`](/docs/intro/concepts/state), and are stored in the Pulumi Service, a file, or a pluggable provider that you choose. These raw values are usually just server names, configuration settings, etc. However, these values sometimes contain sensitive data, such as database passwords or service tokens. **Pulumi never sends authentication secrets or credentials to the Pulumi Service**.
```
